pkg/proc/gdbserial/gdbserver.go

```diff
@@ -922,6 +922,16 @@ continueLoop:
 		}
 	}
 
+	if p.BinInfo().GOOS == "darwin" {
+		images, err := p.conn.getLoadedDynamicLibraries()
+		if err != nil {
+			return nil, stopReason, fmt.Errorf("could not load dynamic libraries %s", err)
```

Comment:
I think it's better to just log this: `logflags.GdbWireLogger().Errorf(...)`

Reply:
5 lines above there is a return statement for ElfUpdateSharedObjects. I believe behaviour for linux and darwin should be the same.